src/porepy/numerics/ad/operators.py

```diff
@@ -1375,22 +1358,20 @@ def __init__(
 
         """
 
-        self._initialize_children()
+        super().__init__(name=name)
+
+    def grid_type(self) -> Literal["subdomains", "interfaces", "boundary grids"]:
```

Comment:
Is GridLikeSequence better, perhaps?

Reply:
Is this about the name GridsLike -> GridLikeSequence? If yes, I like your option more than mine.

Reply:
Sorry about this. My codespace messed up the location of my comments. Yes, that's what I meant.

Reply:
Then I rename it.